task-2/.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
+  ],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 13,
+    sourceType: 'module',
+  },
+  plugins: [
+    'react',
+    'prettier',
```

Comment:
Since you don't yet have a "eslint-plugin-prettier" installed I assume you didn't finish the ESLint+Prettier setup.
Consider running the "npm run lint:fix" command and fixing all the code errors and ESLint+Prettier compatibility issues.

Reply:
Eslint-plugin-prettier - has been installed and is up and running.